src/System.Diagnostics.Process/src/System/Diagnostics/Process.WinRT.cs

```diff
@@ -13,18 +13,18 @@ private bool StartCore(ProcessStartInfo startInfo)
                 : StartWithCreateProcess(startInfo);
         }
 
-        private string GetMainWindowTitle() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private string GetMainWindowTitle() => string.Empty;
 
-        public IntPtr MainWindowHandle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public IntPtr MainWindowHandle => IntPtr.Zero;
 
-        private bool CloseMainWindowCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool CloseMainWindowCore() => false;
 
-        public string MainWindowTitle => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public string MainWindowTitle => string.Empty;
 
-        private bool IsRespondingCore() => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool IsRespondingCore() => true;
 
-        public bool Responding => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        public bool Responding => true;
 
-        private bool WaitForInputIdleCore(int milliseconds) => throw new PlatformNotSupportedException(SR.Win32WindowsNotSupported);
+        private bool WaitForInputIdleCore(int milliseconds) => false;
```

Comment:
> @stephentoub can you be the tie breaker here?

I think we should throw.  WaitForInputIdle() throws even on Windows if the app doesn't have a message loop, e.g. this program
```C#
class Program
{
    static void Main() => System.Diagnostics.Process.GetCurrentProcess().WaitForInputIdle();
}
```
throws:
```
Unhandled Exception: System.InvalidOperationException: WaitForInputIdle failed.  This could be because the process does not have a graphical interface.
   at System.Diagnostics.Process.WaitForInputIdle(Int32 milliseconds)
   at System.Diagnostics.Process.WaitForInputIdle()
   at Program.Main() in c:\users\stoub\Source\Repos\ConsoleApp6\ConsoleApp6\Program.cs:line 3
```